File: IT/1/Funzioni/Esercizi/Java/Esercizio15.java
/* file Esercizio15.java */

public class Esercizio15 {
	static double exp(double b, double ex) {
		return Math.pow(b, ex);
	}

	public static void main(String [] args) {
		System.out.println("Calcolo delle potenze.");

		double b = 1;
		double ex = 2;

		while (ex <= 10) {
			double p = exp(b, ex);
			System.out.println("Potenza di " + b + " elevato a " + ex + " = " + p);
			b++;
			ex++;
		}
	}
}

File: IT/1/Funzioni/Esercizi/C++/esercizio15.cpp
/* file esercizio15.cpp */

#include <iostream>
#include <cmath>

using std::cout;
using std::endl;

double exp(double b, double ex) {
	return pow(b, ex);
}

int main() 
{
	cout << "Calcolo delle potenze." << endl;

	double b = 1;
	double ex = 2;

	while (ex <= 10) {
		double p = exp(b, ex);
		cout << "Potenza di " << b << " elevato a " << ex << " = " << p << endl;
		b++;
		ex++;
	}
}

